### ISSUE TITLE
Missing examples for common next steps

### ISSUE THREAD

**Issue by twelvelabs** [convox-examples/rails#3]
I was able to clone and run this example app locally following the instructions at https://convox.com/docs/getting-started/ and was also able to successfully deploy it to a rack, but was a little lost on how to accomplish what seemed to me to be the next logical steps: using `./bin/rails` to create new models or controllers and running tests.

My first thought was that I should use `convox run`, but that appears to be only for apps deployed to a rack.

The next thought was that I had to use `docker-compose` (which seemed a little counter intuitive), but that failed because the database.yml file is gone and the `DATABASE_URL` isn't defined in the docker compose file... so I figured I probably wasn't on the correct route.

I finally tried running `docker exec -it rails-web bash` to get into the web process started by `convox start`... which again, didn't seem to be super intuitive given how high level everything else in convox is but seemed to work - I was able to run rake and rails commands to initialize the DB and create a new controller. Struck out w/ several different errors attempting to run tests though which I'm starting to suspect is again due to the missing database.yml.

Can y'all please provide documentation for (a) the _right_ way to run rake/rails commands locally, and (b) proper database configuration that will play nice with both `convox start` and running tests?

**Comment by twelvelabs** [comment]
FYI, the `docker exec` approach seems to work fine as long as I add a `database.yml` file back in:

```ruby
default: &default
  adapter: postgresql
  encoding: unicode
  pool: 5
  host: <%= ENV['DATABASE_HOST'] %>
  port: <%= ENV['DATABASE_PORT'] %>
  username: <%= ENV['DATABASE_USERNAME'] %>
  password: <%= ENV['DATABASE_PASSWORD'] %>

development:
  <<: *default
  database: app_development

test:
  <<: *default
  database: app_test

production:
  <<: *default
  url: <%= ENV['DATABASE_URL'] %>
```

This lets me have different DB names for each Rails env.

**Comment by mattmanning** [comment]
Hi @twelvelabs. Thanks a lot for the feedback. We've expanded the README to explain the development workflow with `convox start` a little more thoroughly. Can you please take a look at https://github.com/convox-examples/rails#developing-in-the-container and let us know if you still have questions?

You shouldn't need a `database.yml` file running Rails with Convox. Your database is specified by the `DATABASE_URL` environment variable. When developing locally, `DATABASE_URL` is set by `docker-compose.yml` [links](https://convox.com/docs/linking/). In your deployed apps (staging, production, etc) you set `DATABASE_URL` using the `convox env set DATABASE_URL=...` command.

**Comment by twelvelabs** [comment]
@mattmanning the examples added to the README are exactly what I was looking for. Would be nice to add similar info to the official documentation. I originally was looking at https://convox.com/docs/running-locally/ and https://convox.com/docs/one-off-processes for answers.

Re: `database.yml`, I understand how `DATABASE_URL` is being used but this doesn't work with how Rails executes tests. Rails attempts to clear out whatever database is defined for your test environment before running tests. If I `docker exec` into the web process to run tests, then my development and test environments will be sharing the same `DATABASE_URL`. I was seeing a postgres error when the test runner was trying to drop tables currently in use by the web process... though even if that weren't the case, I wouldn't want to have my dev db constantly reset every time I ran tests locally.

I finally simplified the `database.yml` to the following in order to run tests without issue:

```yaml
production:
  url: <%= ENV['DATABASE_URL'] %>

development:
  url: <%= ENV['DATABASE_URL'] %>

test:
  url: <%= ENV['DATABASE_URL'] %>_test
```

The only other solution I can think of would be to create a `bin/test` wrapper script that manually set a different `DATABASE_URL`.
